portfolio/src/App.css

```diff
@@ -14,18 +21,22 @@
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: #427aa1;
   min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   font-size: calc(10px + 2vmin);
-  color: white;
+  color: #ebf2fa;
 }
 
 .App-link {
-  color: #61dafb;
+  color: #a5be00;
+}
+
+p {
```

Comment:
Not a huge fan of this, but if you do use it, rather than creating a wholly new copy component or anything, move it up the page. Its specificity is quite low, so higher is better.

Reply:
Don't entirely get what you mean here